src/steamship/cli/local_server/server.py

```diff
@@ -0,0 +1,78 @@
+import threading
+from socketserver import TCPServer
+from typing import Optional, Type
+
+from steamship.cli.local_server.handler import make_handler
+from steamship.invocable import Invocable
+
+
+class SteamshipHTTPServer:
+    """A simple HTTP Server that wraps an invocable (package or plugin).
+
+    To use, call:
+
+       server = SteamshipHTTPServer(invocable)
+       server.start()
+
+    To shut down, call:
+
+       server.stop()
+
+    """
+
+    invocable: Type[Invocable]
+    port: int
+    server: TCPServer
+    default_api_key: Optional[str] = (None,)
```

Comment:
Is this tuple syntax necessary for some reason?  Not a big deal, I just want to make sure there's not something here I don't understand